src/test/java/com/inscopemetrics/mad/parsers/TelegrafJsonToRecordParserTest.java

```diff
@@ -37,7 +37,7 @@
 import java.util.Map;
 
 /**
- * Tests for Telegraf JSON format.
+ * Tests for TelegrafJson JSON format.
```

Comment:
Did you mean to have the double JSON here?

Reply:
Auto-refactor. Fixed.